<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>287</y>
     <width>279</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#username
{
background-color: rgba(255, 255, 255, 0.1);
border-radius: 20px;
color: white;
font-size: 16px;
padding: 10px 20px;
font: 10pt &quot;Montserrat&quot;;
}

#username:focus
{
	border: none;
	border: 2px solid #ffffff;
}

</string>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/Login/Images/Login/Background.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>356</y>
     <width>279</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#password
{
background-color: rgba(255, 255, 255, 0.1);
border-radius: 20px;
color: white;
font-size: 16px;
padding: 10px 20px;
font: 10pt &quot;Montserrat&quot;;
}

#password:focus
{
	border: 2px solid #ffffff;
}

</string>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>436</y>
     <width>279</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#loginButton
{
	background-color: #4082f5;
	border-radius: 20px;
	color: white;
	font-size: 16px;
	font: 12pt &quot;Montserrat&quot;;
}

#loginButton:hover
{
	background-color: #5a96ff;
}

#loginButton:pressed
{
	border: 2px solid #ffffff;
}</string>
   </property>
   <property name="text">
    <string>SIGN IN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>364</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton
{
 	background-color: transparent;
	border-image: url(:/Login/Images/Login/hide.png);
	background: none;
	border: none;
	background-repeat: none;
}

#pushButton:pressed
{
	border-image: url(:/Login/Images/Login/view.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorUsername">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>329</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #f44336;
font: 12pt &quot;Montserrat&quot;;
qproperty-alignment: AlignCenter;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorPassword">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>398</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #f44336;
font: 12pt &quot;Montserrat&quot;;
qproperty-alignment: AlignCenter;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="signIn">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>223</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#signIn
{
 	background-color: transparent;
	background: none;
	border: none;
	background-repeat: none;
	color: white;
	font: 16pt &quot;Montserrat&quot;;
}

#signIn:hover
{
	color: rgba(255, 255, 255, 1);
}</string>
   </property>
   <property name="text">
    <string>SIGN IN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signOut">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>223</y>
     <width>94</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#signOut
{
 	background-color: transparent;
	background: none;
	border: none;
	background-repeat: none;
	color: rgba(255, 255, 255, 0.4);
	font: 16pt &quot;Montserrat&quot;;
}

#signOut:hover
{
	color: rgba(255, 255, 255, 1);
}</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QLabel" name="underline">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>250</y>
     <width>82</width>
     <height>3</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/Login/Images/Login/underline.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="underline2">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>250</y>
     <width>88</width>
     <height>3</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/Login/Images/Login/underline2.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_2">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>304</y>
     <width>279</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#username_2
{
background-color: rgba(255, 255, 255, 0.1);
border-radius: 20px;
color: white;
font-size: 16px;
padding: 10px 20px;
font: 10pt &quot;Montserrat&quot;;
}

#username_2:focus
{
	border: 2px solid #ffffff;
}

</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>285</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Montserrat&quot;;
color: rgba(255, 255, 255, 0.7);</string>
   </property>
   <property name="text">
    <string>USERNAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_2">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>383</y>
     <width>279</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#password_2
{
background-color: rgba(255, 255, 255, 0.1);
border-radius: 20px;
color: white;
font-size: 16px;
padding: 10px 20px;
font: 10pt &quot;Montserrat&quot;;
}

#password_2:focus
{
	border: 2px solid #ffffff;
}

</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>364</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Montserrat&quot;;
color: rgba(255, 255, 255, 0.7);</string>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="retype_password">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>463</y>
     <width>279</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#retype_password
{
background-color: rgba(255, 255, 255, 0.1);
border-radius: 20px;
color: white;
font-size: 16px;
padding: 10px 20px;
font: 10pt &quot;Montserrat&quot;;
}

#retype_password:focus
{
	border: 2px solid #ffffff;
}

</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="retypeLabel">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>444</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Montserrat&quot;;
color: rgba(255, 255, 255, 0.7);</string>
   </property>
   <property name="text">
    <string>RE-ENTER PASSWORD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>539</y>
     <width>279</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#registerButton
{
	background-color: #4082f5;
	border-radius: 20px;
	color: white;
	font-size: 16px;
	font: 12pt &quot;Montserrat&quot;;
}

#registerButton:hover
{
	background-color: #5a96ff;
}

#registerButton:pressed
{
	border: 2px solid #ffffff;
}</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>361</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/Login/Images/Login/Image-1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="wrongUsername">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>313</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/Login/Images/Login/error.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="wrongPassword">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>394</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/Login/Images/Login/error.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="wrongPassword_2">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>473</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/Login/Images/Login/error.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="validUsername">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>313</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/Login/Images/Login/success.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="validPassword">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>394</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/Login/Images/Login/success.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="validPassword_2">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>473</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/Login/Images/Login/success.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>username</zorder>
  <zorder>password</zorder>
  <zorder>loginButton</zorder>
  <zorder>pushButton</zorder>
  <zorder>errorUsername</zorder>
  <zorder>errorPassword</zorder>
  <zorder>signIn</zorder>
  <zorder>signOut</zorder>
  <zorder>underline</zorder>
  <zorder>underline2</zorder>
  <zorder>username_2</zorder>
  <zorder>usernameLabel</zorder>
  <zorder>password_2</zorder>
  <zorder>passwordLabel</zorder>
  <zorder>retype_password</zorder>
  <zorder>retypeLabel</zorder>
  <zorder>registerButton</zorder>
  <zorder>label</zorder>
  <zorder>wrongUsername</zorder>
  <zorder>wrongPassword</zorder>
  <zorder>wrongPassword_2</zorder>
  <zorder>validUsername</zorder>
  <zorder>validPassword</zorder>
  <zorder>validPassword_2</zorder>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
